src/Nethermind/Nethermind.Blockchain/Processing/BlockchainProcessor.cs

```diff
@@ -281,7 +282,10 @@ public Block Process(Block suggestedBlock, ProcessingOptions options, IBlockTrac
 
             ProcessingBranch processingBranch = PrepareProcessingBranch(suggestedBlock, options);
             PrepareBlocksToProcess(suggestedBlock, options, processingBranch);
+
+            Stopwatch stopwatch = Stopwatch.StartNew();
```

Comment:
Why not reuse a stopwatch?

Reply:
it is super light

Reply:
oh, it is a class :)

Reply:
cannot reuse it as we have multiple threads accessing this, would have to be threadlocal - better just create a light object which will be GCed after each block (not much)